website/docs/r/codeartifact_domain.html.markdown

```diff
@@ -13,13 +13,8 @@ Provides a CodeArtifact Domain Resource.
 ## Example Usage
 
 ```hcl
-resource "aws_kms_key" "example" {
-  description = "domain key"
-}
-
 resource "aws_codeartifact_domain" "example" {
   domain         = "example"
-  encryption_key = aws_kms_key.example.arn
```

Comment:
Most resources seem to prefer the AWS managed keys by default in their docs. Let me know if you want a separate example for providing a custom key (the stuff in the red text here).

Reply:
Looks good this way.